lib/make/makefile.blast

```diff
@@ -19,6 +19,17 @@ SHELL := /bin/bash
 NCBI_BLAST_DB_PATH  = YOU NEED TO SET THE NCBI_BLAST_DB_PATH VARIABLE (see makefile.blast)
 BLAST_PROTEIN_DB    = YOU NEED TO SET THE BLAST_PROTEIN_DB VARIABLE (see makefile.blast)
 BLAST_NUCLEOTIDE_DB = YOU NEED TO SET THE BLAST_NUCLEOTIDE_DB VARIABLE (see makefile.blast)
+BLAST_NUCLEOTIDE_DB_PATH = .
```

Comment:
Why do you need this (and the next)? Can't you use the BLAST_NUCLEOTIDE_DB variable?

Reply:
Agreed, will change accordingly.